Index: pjmedia/src/pjmedia/transport_srtp.c
===================================================================
--- pjmedia/src/pjmedia/transport_srtp.c	(revision 5500)
+++ pjmedia/src/pjmedia/transport_srtp.c	(revision 5499)
@@ -1418,8 +1418,6 @@
 
 	/* Generate crypto attribute if not yet */
 	if (pjmedia_sdp_media_find_attr(m_loc, &ID_CRYPTO, NULL) == NULL) {
-	    int tag = 1;
-
 	    /* Offer only current active crypto if any, otherwise offer all
 	     * crypto-suites in the setting.
 	     */
@@ -1434,7 +1432,7 @@
 		buffer_len = MAXLEN;
 		status = generate_crypto_attr_value(srtp->pool, buffer, &buffer_len,
 						    &srtp->setting.crypto[i],
-						    tag);
+						    i+1);
 		if (status != PJ_SUCCESS)
 		    return status;
 
@@ -1444,7 +1442,6 @@
 		    attr = pjmedia_sdp_attr_create(srtp->pool, ID_CRYPTO.ptr,
 						   &attr_value);
 		    m_loc->attr[m_loc->attr_count++] = attr;
-		    ++tag;
 		}
 	    }
 	}
